introduction.md

```diff
@@ -3,3 +3,12 @@
 Repositorio forkeado para la realizacion del curso GitHub en CodeSchool.
 
 My favorite Code School path: Ruby, JavaScript.
+
+## Skills: 
+
+* Ruby
+* Javascript
+* Rails
+* Git
+
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?